flatpak-preinstall-ftp.ks.in

```diff
@@ -0,0 +1,22 @@
+#
+# Check Flatpak preinstall.d  support in Anaconda using FTP.
+
+%ksappend repos/default-ftp.ks
+%ksappend common/common_no_payload.ks
+
+# Install RPM marking firefox in Flatpak for installation.
+# The flatpak have to be installed from the main repository (BaseOS) where have to be
+# ./flatpaks directory.
```

Comment:
I find this comment confusing.
The directory name is "Flatpaks", not "flatpaks".
Also, the directory is not part of the BaseOS repository, it's on the same level as BaseOS or AppStream repositories.

Reply:
I've tried to improve this. Is it better now?